Assets/_Standard Assets/Characters/Scripts/Physics/OpenCharacterController.cs

```diff
@@ -2184,5 +2198,11 @@ void SetRootToOffset()
 				m_PlayerRootTransform.localPosition = m_RootTransformOffset;
 			}
 		}
+
+		// Update values that are used for computing slow down against walls.
+		void UpdateSlowAgainstWalls()
```

Comment:
You're right that this name is a bit awkward. I actually just think this method is unnecessary. It's only one line, and only _really_ needs to happen in Awake. I think just inline the code everywhere it's called, and remove this.